Sources/Logging/Locks.swift

```diff
@@ -66,11 +72,12 @@ internal final class Lock {
     deinit {
         #if os(Windows)
         // SRWLOCK does not need to be free'd
+        self.mutex.deallocate()
```

Comment:
unsure why the Windows branch is touched here?

Reply:
I originally had another case for WASI here (which had no mutex), the change is not necessary anymore anymore since I then excluded the whole Lock class.